### PR TITLE
User challenges list styling

### PR DESCRIPTION
**Pull request by prasadwalvekar**
### Changes

- Add styles for User Challenges List screen

### Screenshot

Screenshot 2021-08-10 at 10 35 04 PM

### CODE REVIEW

File: packages/app/src/components/profile/UserChallengesList.tsx [hunk 1]
```diff
@@ -224,75 +224,121 @@ export const UserChallengesList = (props: Props): React.ReactElement => {
...
-        </View>
+        </Gradient>
     );
 };
 
 const styles = StyleSheet.create({
+    title: {
+        textAlign: 'center',
+        fontSize: 34,
+        margin: '5% 0%'
```
Comment (stormcloud266) [inline packages/app/src/components/profile/UserChallengesList.tsx:298]: This syntax causes an error for me. `marginVertical: '5%'` would work here.

File: packages/app/src/components/profile/UserChallengesList.tsx [hunk 2]
```diff
@@ -224,75 +224,121 @@ export const UserChallengesList = (props: Props): React.ReactElement => {
...
+        flex: 1
+    },
+    background: {
+        paddingHorizontal: '5%',
+        marginTop: '15%',
+        borderColor: 'rgba(251, 250, 250, 0.7)',
+        borderWidth: 1,
+        borderTopLeftRadius: 10,
+        borderTopRightRadius: 10,
+        backgroundColor: 'rgba(255, 255, 255, 0.5)'
```
Comment (stormcloud266) [inline packages/app/src/components/profile/UserChallengesList.tsx:325]: Let's add `flex: 1` here to pull the background all the way done even without any list items. I saw that I missed this in the market list as well and will be updating it. 

Screen Shot 2021-08-10 at 12 11 03 PM

**Review by stormcloud266 — CHANGES_REQUESTED** [review]
Everything looks really good! Thank you.

**Comment by prasadwalvekar** [comment]
Code review changes done.

**Review by stormcloud266 — APPROVED** [review]

**Pull request merged**